src/main/kotlin/frc/robot/subsystems/shooter/ShooterConstants.kt

```diff
@@ -0,0 +1,86 @@
+package frc.robot.subsystems.shooter
+
+import com.ctre.phoenix6.configs.FeedbackConfigs
+import com.ctre.phoenix6.configs.MotorOutputConfigs
+import com.ctre.phoenix6.configs.Slot0Configs
+import com.ctre.phoenix6.configs.TalonFXConfiguration
+import com.ctre.phoenix6.signals.InvertedValue
+import edu.wpi.first.units.*
+import frc.robot.lib.webconstants.LoggedTunableNumber
+
+object ShooterConstants {
+    const val GEAR_RATIO_TOP: Double = 1.0
+    const val GEAR_RATIO_BOTTOM: Double = 1.0
+
+    val TOP_ROLLER_TOLERANCE: Measure<Velocity<Angle>> = Units.RotationsPerSecond.of(1.0)
+    val BOTTOM_ROLLER_TOLERANCE: Measure<Velocity<Angle>> = Units.RotationsPerSecond.of(1.0)
+    val MOMENT_OF_INERTIA_TOP: MutableMeasure<Mult<Mult<Mass, Distance>, Distance>> =
+        Units.Kilograms.mult(Units.Meters).mult(
+            Units.Meters
+        ).of(0.0008).mutableCopy()
+    val MOMENT_OF_INERTIA_BOTTOM: MutableMeasure<Mult<Mult<Mass, Distance>, Distance>> =
+        Units.Kilograms.mult(Units.Meters).mult(Units.Meters).of(0.0008).mutableCopy()
```

Comment:
I would add a few extension functions for `Measure` of basic arithmetic operations.